composer.json

```
@@ -32,12 +36,13 @@
     "drush/drush": "^8",
     "drupal/drupal-extension": "~3.4",
     "drupal/devel": "*",
-    "openeuropa/task-runner": "^0.4",
-    "drupal/xautoload": "*"
+    "drupal/xautoload": "*",
+    "openeuropa/task-runner": "^0.4"
   },
   "autoload": {
     "psr-4": {
-      "Drupal\\Tests\\registryonsteroids\\": "./tests/src/"
+      "Drupal\\Tests\\registryonsteroids\\": "./tests/src/",
+      "Drupal\\registryonsteroids\\": "./src/"
```

Adding this line enables composer do the autoload for the files in src/.
This means we could now disable xautoload and tests would still pass, thus potentially hiding failures.

Is this change needed for anything?

---

This is only needed for the tests, it's not used in D7.